package.json

```diff
@@ -40,13 +40,15 @@
     "react": "16.x.x"
   },
   "dependencies": {
+    "@emotion/core": "^10.0.10",
```

Comment:
Can this be made a devDependency or a peerDependency? I hate to drag all of emotion into a pure React project.

Reply:
Related - do you think we should split out "src/components" into its own package?

Reply:
It's not used at all, so removed it entirely.

Reply:
> Related - do you think we should split out "src/components" into its own package?

Not opinionated on that, but having a `formtron-components` pick with all them listed as separate components might be a good idea.

Reply:
I guess the main reason to leave them in the same repo is because without them... you basically wouldn't have anything to look at in Storybook. And if we're keeping them in the same repo, then the choices are: monorepo with two packages (which complicates publishing them) or just publish them as the same package. And inevitably I always come back to "well, its so much easier to just publish them as one package. I guess as long as the unshared dependencies between the two don't get too big it doesn't hurt anything - although it increases the likelihood that something in Formtron might accidentally depend on something in ui-kit.

I guess even if that happened, it's not the end of the world. Mostly just conceptual purity / my ego that's dividing the two projects - there's no practical reason to. :)